src/post/post.service.ts

```diff
@@ -541,4 +544,56 @@ export class PostService {
         await this.postRepository.save(post);
         return ok(true);
     }
+
+    async updateGhostPostByUuid(
+        account: Account,
+        ghostPost: GhostPost,
+    ): Promise<Result<boolean, UpdatePostError>> {
+        const post = await this.postRepository.getByUuid(ghostPost.uuid);
+        if (post === null) {
+            return error('post-not-found');
+        }
+        const postResult = await Post.createArticleFromGhostPost(
```

Comment:
I don't think we want to be using `createArticleFromGhostPost` like this 🤔 I would expect that we would have a method like `updateFromGhostPost` on the entity, which would handle all of the logic that is below like deleting etc...